feconf.py

```diff
@@ -736,6 +736,9 @@ def get_empty_ratings():
 # Page mode for the search results page.
 LIBRARY_PAGE_MODE_SEARCH = 'search'
 
+# group_name param for GET request in Splash.js
+SPLASH_PAGE_FEATURED = 'splash_page_featured'
```

Comment:
feconf.LIBRARY_CATEGORY_SPLASH_PAGE_FEATURED

(and put it near the other similar constants)